kernel/idt.c

```diff
@@ -21,6 +21,14 @@ struct IDT_entry *idt;
 
 void create_idt () {
   idt = (struct IDT_entry *)allocate_phys_page();
+  const struct {
```

Comment:
No change required, but this is one of the strangest lines of C I've seen in a long time.